test/parallel/test-cli-node-options.js

```diff
@@ -60,7 +60,12 @@ if (common.hasCrypto) {
 }
 
     // V8 options
+expect('--abort-on-uncaught-exception', 'B\n');
```

Comment:
nit: define `const printB_expectedOutput = 'B\n';` (Actualy I think `'^B\n$'` would be better).